flagsmith/resource_flag.go

```diff
@@ -48,7 +48,7 @@ func (t flagResourceType) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Dia
 					"type": {
 						Type:                types.StringType,
 						MarkdownDescription: "Type of the feature state value, can be `unicode`, `int` or `bool`",
-						Required:            true,
```

Comment:
This was implicitly optional earlier, i.e: if the parent block is optional direct children of that will be considered optional, but that's not the case with the recent update, hence making it explicitly optional (notice no change in documentation, i.e: no change in api)